src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1568,6 +1595,48 @@ getTestNameDescriptions(bool addAdditionOptions)
   return reportDescriptionCollection;
 }
 
+/*
+ * Extended keys helper struct
+ */
+struct ExtendedKeysStruct {
+  std::string test_name;
+  std::string param_name;
+  std::string description;
+};
+
+static std::vector<ExtendedKeysStruct>  extendedKeysCollection = {
+  {"dma", "block-size", "Memory transfer size (bytes)"}
```

Comment:
Not sure if it is a good idea use implicit constructors for named members. If the struct is ever changed this could cause a lot of issues. You can make a constructor for the struct and pass it in that way. There is an alternative where you say ".test_name = "dma, .param_name = "block_size" etc. but it is compiler dependent (and I our windows compiler in the pipeline does not like that)